docs/QuickStart.md

```diff
@@ -0,0 +1,154 @@
+# QuickStart
+
+## Installation
+
+Two ways for installation: you can get NNI from pypi or clone the repo and install it from there.
+
+**Install through pip**
+
+* We support Linux and MacOS in current stage, Ubuntu 16.04 or higher, along with MacOS 10.14.1 are tested and supported. Simply run the following `pip install` in an environment that has `python >= 3.5`.
+
+```bash
+    python3 -m pip install --user --upgrade nni
+```
+
+* Note:
+  * If you are in docker container (as root), please remove `--user` from the installation command.
+  * If there is any error like `Segmentation fault`, please refer to [FAQ][1]
+
+**Install through source code**
+
+* We support Linux (Ubuntu 16.04 or higher), MacOS (10.14.1) in our current stage. 
+* Run the following commands in an environment that has `python >= 3.5`, `git` and `wget`.
+
+```bash
+    git clone -b v0.4.1 https://github.com/Microsoft/nni.git
+    cd nni
+    source install.sh
+```
+
+For the system requirements of NNI, please refer to [Install NNI][2]
+
+## First "Hello World" experiment: MNIST
+
+Now let's start to run our first NNI experiment, here is a minimal experiment we prepared to teach you how to using NNI:
+
+```yaml
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+trainingServicePlatform: local
+searchSpacePath: search_space.json
+useAnnotation: false
+tuner:
+  builtinTunerName: TPE
+trial:
+  command: python3 mnist.py
+  codeDir: .
+  gpuNum: 0
+```
+
+**We did two things in this file:**
+
+* Indicates the path of [search_space.json][3], which specify the `range of hyper-paramaters` to be searched.
+* Define a NNI trial in [mnist.py][4], an individual `attempt` at applying a set of parameters on a model.
+
+Note: other configurations can be kept by their default value. We will talk about them in the following tutorial.
+
+* Everything is ready now! **The experiment can be run from the command-line**:
+
+```bash
+    nnictl create --config nni/examples/trials/mnist/config.yml
+```
+
+* Wait for the message `INFO: Successfully started experiment!` in the command line. This message indicates that your experiment has been successfully started. And this is what we expected to get:
+
+```
+INFO: Starting restful server...
+INFO: Successfully started Restful server!
+INFO: Setting local config...
+INFO: Successfully set local config!
+INFO: Starting experiment...
+INFO: Successfully started experiment!
+-----------------------------------------------------------------------
+The experiment id is egchD4qy
+The Web UI urls are: http://223.255.255.1:8080   http://127.0.0.1:8080
+-----------------------------------------------------------------------
+
+You can use these commands to get more information about the experiment
+-----------------------------------------------------------------------
```

Comment:
could you please help update the view of command information?
```
-----------------------------------------------------------------------
         commands                       description
1. nnictl experiment show        show the information of experiments
2. nnictl trial ls               list all of trial jobs
3. nnictl top                    monitor the status of running experiments
4. nnictl log stderr             show stderr log content
5. nnictl log stdout             show stdout log content
6. nnictl stop                   stop an experiment
7. nnictl trial kill             kill a trial job by id
8. nnictl --help                 get help information about nnictl
-----------------------------------------------------------------------
```

Reply:
Thanks for point out!!